### ISSUE TITLE
M1 arm64 support

### ISSUE THREAD

**Issue by craftzdog** [electron/mksnapshot#35]
Hi,

I tried to install it on my M1 macbook but got an error:

```
> npm config set arch arm64
> npm i -D electron-mksnapshot@latest
...
WARNING: mksnapshot does not run on arm64. Download
  https://github.com/electron/electron/releases/download/v11.0.0/mksnapshot-v11.0.0-darwin-arm64-x64.zip
  on a x64 darwin OS to generate arm64 snapshots.
```

The given URL seems like not existing. I believe that Electron@11 supports M1 hardwares.
How can I solve this?
Thanks in advance.

**Comment by jkleinsc** [comment]
@craftzdog I just published electron-mksnapshot v11.0.1 which fixes this issue.

**Comment by craftzdog** [comment]
Thank you, @jkleinsc!
